lib/simctl.js

```diff
@@ -523,38 +524,39 @@ async function getDevices (forSdk = null) {
      *   }
      * }
      */
-    devices = {};
-    for (let [sdkName, entries] of _.toPairs(JSON.parse(stdout).devices)) {
-      if (sdkName.indexOf('iOS') !== 0) {
+    for (const [sdkName, entries] of _.toPairs(JSON.parse(stdout).devices)) {
+      const versionMatch = new RegExp(`^${_.escapeRegExp(platform)}\\s+(\\S+)`, 'i').match(sdkName);
```

Comment:
No need to create the same regular expression each time through the loop.

Reply:
yep